### DISCUSSION TITLE
Unable to generate documenatation in ubuntu.

### DISCUSSION THREAD

**Discussion by bhagath555** [jnbrunet/caribou#141] [Q&A]
Hello Caribou Community,
I am a beginner Caribou coder, from India, I would like to generate code documentation in ubuntu OS. As suggested in the doc/ directory README.md, I tried to install in below packages
1. `texlive-scheme-basic`
2. `texlive-epstopdf`
3. `texlive-newunicodechar`

But, there is no luck in installing these packages using `sudo apt install package_name_here` commands. 
I installed DOXYGEN and SPHINX, but about unable to install these packages. Is there any other way to install above mentioned texlive packages. Please provide the detailed answer.
Thanks in advance.
Note : I installed all the other necessary packages that are required for Caribou and SOFA installation.

**Comment by jnbrunet** [comment]
Hey @bhagath555 ,

Sorry for the late reply. Did you try with docker? This way you don't have to bother with installing packages on your PC.
